modules/User Admin/user_manage_edit.php

```diff
@@ -36,7 +37,9 @@
 
     $returns = array();
     $returns['warning1'] = __('Your request was completed successfully, but one or more images were the wrong size and so were not saved.');
-    $page->return->addReturns($returns);
+    if (isset($_GET['return'])) {
```

Comment:
This seems to be undoing some OOification work...?

Reply:
Oh! Must have accidentally reverted that when rebasing, will push the fix post-merge. Thanks!